browser/help/topics/v3-ht-annotations.md

```diff
@@ -165,22 +165,22 @@ The gnomAD v3 Hail Table annotations are defined below:
-    - **raw_score**: Raw CADD scores are interpretable as the extent to which the annotation profile for a given variant suggests that the variant is likely to be 'observed' (negative values) vs 'simulated' (positive values). Larger values are more deleterious.
-  - **sift**:
-    - **sift_max**: A normalized probability of observing the new amino acid at that position to affect protein function, and ranges from 0 to 1. Similar to what we did for REVEL and PrimateAI-3D, we will only show max scores from MANE Select and Canonical transcripts. 
-  - **polyphen**:
-    - **polyphen_max: Prediction of the possible impact of an amino acid substitution on the structure and function of a human protein, ranging from 0.0 (tolerated) to 1.0 (deleterious). We also only show predictions of a max score from MANE Select and canonical transcripts. 
-  - **phylop**:
-    - **phylop**: Base-wise conservation of Zoonomia 241 placental mammals, ranging from -20 to 9.28, phyloP can measure acceleration (faster evolution than expected under neutral drift, assigned negative scores) as well as conservation (slower than expected evolution, assigned positive scores). 
-  - **splice_ai**:
-    - **max_ds**: The maximum delta score across 4 splicing consequences across genes if a variant was predicted on multiple genes, interpreted as the probability of the variant being splice-altering. Ranging from 0 to 1, delta score > 0.2 has been used as a permissive threshold for retaining high sensitivity. 
-  - **pangolin**:
-    - **largest_ds**: The largest delta score of a variant involved in 2 splicing consequences across genes if a variant was predicted on multiple genes. It ranges from -1 to 1, positive are for splice loss variants and > 0 are for splice-gain variants. Pangolin also proposed a cutoff of >|0.2| for the predicted increase or decrease in splice site usage. 
+- **in_silico_predictors**: Variant prediction annotations. We put together the prediction scores from multiple in silico predictors for variants that are predicted to be missense, impacting protein function, evolutionarily conserved or splice-altering. We focus on MANE Select and canonical transcripts if the prediction scores are available for multiple transcripts.
+  - **revel**: Rare Exome Variant Ensemble Learner (REVEL) [score](https://sites.google.com/site/revelgenomics/about). An ensemble score for predicting the pathogenicity of missense variants (based on 13 other variant predictors).
+    - **revel_max**: Score predicting the pathogenicity of missense variants with a range of 0 to 1. Variants with higher scores are predicted to be more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
+  - **primateai_3d**: A deep neural network-trained [score](https://www.science.org/doi/10.1126/science.abn8197) for classifying pathogenicity of missense variants.
```

Comment:
we also don't know if we'll have this score for v4 MVP yet

Reply:
Hopefully we will, we should hear from them next week or so.

Reply:
leaving this comment open in case we need to remove this section